# Use cmake-variable GIT_REVISON value, and do not look git revision number using git command
# compilation of reorder.f90 crashes intel compiler with -O2
# Obey CMAKE_INSTALL_PREFIX
# July 10th 2018 by B. Hajgato (Free University Brussels - VUB)
--- cmake/custom/git.cmake.orig	2018-04-20 09:44:47.000000000 +0200
+++ cmake/custom/git.cmake	2018-07-10 14:06:50.949238887 +0200
@@ -1,12 +1,3 @@
-set(GIT_REVISION)
-find_package(Git)
-if (GIT_FOUND)
-    execute_process(
-        COMMAND ${GIT_EXECUTABLE} rev-list --abbrev-commit --max-count=1 HEAD
-        OUTPUT_VARIABLE GIT_REVISION
-        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-        )
     string(STRIP
         ${GIT_REVISION}
         GIT_REVISION)
-endif()
--- src/libgimic/CMakeLists.txt.orig	2018-04-20 09:44:47.000000000 +0200
+++ src/libgimic/CMakeLists.txt	2018-07-10 15:17:24.542709076 +0200
@@ -92,4 +92,4 @@
 install(FILES GimicInterface.h gimic_interface.h gimic.pxd
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
     )
-
+SET_SOURCE_FILES_PROPERTIES(reorder.f90 PROPERTIES COMPILE_FLAGS "${COMPILE_FLAGS} -O1")
--- cmake/custom/own.cmake.orig	2018-04-20 09:44:47.000000000 +0200
+++ cmake/custom/own.cmake	2018-07-10 16:08:05.999455805 +0200
@@ -1,4 +1,4 @@
-set(CMAKE_INSTALL_PREFIX ${PROJECT_BINARY_DIR})
+#set(CMAKE_INSTALL_PREFIX ${PROJECT_BINARY_DIR})
 
 include(GNUInstallDirs)
 
